llpc/test/shaderdb/core/LocalInvocationIdSwizzle.spvasm

```diff
@@ -0,0 +1,108 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -v %gfxip %s --force-compute-shader-thread-id-swizzling | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} pipeline before-patching results
+; SHADERTEST: %{{[0-9]+}} = call <3 x i32> @lgc.swizzle.local.invocation.id(<3 x i32> %{{[0-9]+}})
+; SHADERTEST: AMDLLPC SUCCESS
+; END_SHADERTEST
+; BEGIN_SHADERTEST
+; RUN: amdllpc -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
```

Comment:
The two cases need to use different check prefix instead of SHADERTEST twice.

SHADERTEST has no special meaning, it's just a convention in llpc to use it when there's a single RUN line. See  https://llvm.org/docs/CommandGuide/FileCheck.html#the-filecheck-check-prefix-option.

Reply:
Thanks for your correction. I will update the check-prefix.